enarx.ks

```diff
@@ -70,6 +73,14 @@ zram
 %end
 
 %post
+# Give SGX and SEV device node access to their respective groups
+cat > /etc/udev/rules.d/50-sgx.rules <<EOF
+SUBSYSTEM=="sgx", KERNEL=="sgx/provision", GROUP="sgx", MODE="0660"
+SUBSYSTEM=="sgx", KERNEL=="sgx/enclave", MODE="0666"
```

Comment:
I am still reading through the patches to see if there's a hint as to why this is the recommended mode for this node. I can change this to `0660` if we'd rather err on something less permissive and see if it gets in the way later (which I'm not entirely sure why it would off the top of my head)

Reply:
I'm happy to follow Intel's guidance here.